agent/api.go

```diff
@@ -36,6 +36,8 @@ func (a *agent) apiHandler() http.Handler {
 		cacheDuration: cacheDuration,
 	}
 	r.Get("/api/v0/listening-ports", lp.handler)
+	r.Get("/debug/magicsock", a.HandleHTTPDebugMagicsock)
+	r.Get("/debug/magicsock/debug-logging/{state}", a.HandleHTTPMagicsockDebugLoggingState)
```

Comment:
nit: `/debug/magicsock/debug...` sounds weird. Maybe POST `/debug/magicsock/enable-logger` instead?

Reply:
I'd rather keep this the same for consistency purposes

Reply:
Ok